manifest/oci_index.go

```diff
@@ -12,6 +12,13 @@ import (
 	imgspecv1 "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
+const (
+	// Following annotation is used while performing instance selection.
+	// If more than one image exists for the same platform then the image
+	// compressed with `zstd` will be given priority.
+	ZSTDInstanceAnnotation = "io.github.containers.compression.zstd"
```

Comment:
This either shouldn’t be a part of the public API, or if we want to make a Go constant, it should be in a separate subpackage (`c/image/manifest/compression`?) so that it is usable without dragging in all of the dependencies of `c/image/copy`.

---

Also, the name (together with the package name) should be sufficiently descriptive. “instance” of what? But let’s figure out the public/nonpublic decision first.

Reply:
Made a new const `OCI1InstanceAnnotationCompressionZSTD`